crud/stats.go

```diff
@@ -0,0 +1,41 @@
+package crud
+
+import (
+	"context"
+
+	"github.com/tarantool/go-tarantool"
+)
+
+// StatsRequest helps you to create request object to call `crud.stats`
+// for execution by a Connection.
+type StatsRequest struct {
+	spaceRequest
```

Comment:
How does space request works? `space` is an optional argument for `crud.stats`, but is a required one for any other request.

Reply:
Fixed.